tutoraspects/templates/openedx-assets/assets/datasets/clickhouse_computed_metrics.yaml

```diff
@@ -0,0 +1,117 @@
+_file_name: clickhouse_computed_metrics.yaml
+table_name: clickhouse_computed_metrics
+main_dttm_col: null
+description: null
+default_endpoint: null
+offset: 0
+cache_timeout: null
+schema: system
+sql: |
+    SELECT
+      name,
+      toFloat64(value) AS value,
+      description
+    FROM system.metrics
+    WHERE metric IN (
+      'Query',
+      'DelayedInserts',
+      'DistributedFilesToInsert'
+    )
+
+    UNION ALL
+
+    SELECT
+      metric AS name,
+      round(toFloat64(value), 2) AS value,
+      description
+    FROM system.asynchronous_metrics
+    WHERE metric IN ('MaxPartCountForPartition', 'Uptime')
+
+    UNION ALL
+
+    SELECT
+      name,
+      toFloat64(value) AS value,
+      description
+    FROM system.events
+    WHERE event IN (
+      'RejectedInserts',
+      'ReplicatedDataLoss',
+      'DataAfterMergeDiffersFromReplica',
+      'DataAfterMutationDiffersFromReplica'
+    )
+
+    UNION All
+
+    SELECT
+      'StuckReplicationTasks' AS name,
+      toFloat64(count()) AS value,
+      'Replication tasks that were retried or postponed over 100 times.' AS description
```

Comment:
I'm somewhat concerned that we won't be able to translate these descriptions, but we have another task to look at how to make that work so it's non-blocking.